src/core/config_mod.F90

```diff
@@ -3,6 +3,7 @@
 !!
 !! This module contains subroutines and functions related to the ConfigType instance of CATChem.
```

Comment:
Can we use a different word instead of "instance"? "derived type" is the official term, but "structure" or "data type" could be fine too.